src/components/dockerRegistry/Docker.tsx

```diff
@@ -665,6 +794,11 @@ const DockerForm = ({
                                   : ignoredClusterIdsCsv,
                       }
                     : null,
+            remoteConnectionConfig: getRemoteConnectionConfig(
+                customState.remoteConnectionConfig,
+                customState.remoteConnectionConfig.connectionMethod.value,
```

Comment:
why are we sending the 2nd param. it can be extracted from the first. please check

Reply:
we need it to make it generalized for both docker and cluster